### PR TITLE
docs: Add interpret tutorial with Transformers

### LINKED ISSUE #1729: Add tutorial about using model explanations

### PR DESCRIPTION
**Pull request by dvsrepo**
Closes #1729

### CODE REVIEW

**Comment by dvsrepo** [comment]
closes #1729

**Comment by frascuchon** [comment]
Please @dvsrepo move the linked issue to `Pending QA`, set it the proper milestone (v0.18.0 if applicable), and adapt the PR title to the changelog messages `docs: ....`

**Review by frascuchon — CHANGES_REQUESTED** [review]
Could you revert changes on files affected only by the python engine diffs?

File: docs/tutorials/weak-supervision-multi-label.ipynb [hunk 1]
```diff
@@ -1937,7 +1937,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3",
```
Comment (frascuchon) [inline docs/tutorials/weak-supervision-multi-label.ipynb]: could you revert this?...

**Review by frascuchon — APPROVED** [review]

**Pull request merged**